client/components/organization-wrapper/organization-wrapper.test.js

```diff
@@ -214,7 +212,7 @@ describe("Test <OrganizationWrapper /> routes", () => {
 
   it("should display status if authenticated", () => {
     wrapper = mountComponent(props, ["/default/status"]);
-    expect(mapRoutes(wrapper)["/default/status"]).toBe(Suspense);
+    expect(mapRoutes(wrapper)["/default/status"]).toBe(undefined);
```

Comment:
```suggestion
    expect(mapRoutes(wrapper)["/default/status"]).toBe(Suspense);
```
This is not giving Suspense when I tried. Do you know how to fix this?
Also, the component.find(Routes) returns empty but routes are present in that.
This is the only test where I faced problems in fixing.